infrastructure/lib/utils/addDefaultTags.ts

```diff
@@ -1,5 +1,6 @@
 import { Tags, Resource, Stack } from "aws-cdk-lib";
 
-export default function addAWSApplicationTag(resource: Resource | Stack) {
+export default function addDefaultTags(resource: Resource | Stack) {
   Tags.of(resource).add("awsApplication", process.env.AWS_APPLICATION_TAG as string);
+  Tags.of(resource).add("appName", "api-designer");
```

Comment:
another implementation for a common concern addressed by this PR: make the resources easily identifiable